tests/integration-tests/tests/nccl/test_nccl/test_nccl/nccl_tests_submit_openmpi.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#SBATCH --nodes=2
+#SBATCH --exclusive
+#SBATCH --ntasks-per-node=8
+
+
+module load openmpi
+NCCL_VERSION='2.7.8-1'
```

Comment:
You should change this version whenever NCCL is upgraded in pcluster AMIs.

Reply:
We are currently not installing NCCL in pcluster AMIs; that's why we do it in this test. Anyway, I agree we should find a way to always use the latest version. For now, versions are parameterised exactly for this reason.